scirpy/_tools/_diversity.py

```diff
@@ -12,11 +13,17 @@ def alpha_diversity(
     groupby: str,
     *,
     target_col: str = "clone_id",
+    method: str = "normalized shannon entropy",
     inplace: bool = True,
     key_added: Union[None, str] = None,
```

Comment:
Maybe worth passing `**kwargs` to the method function? Would allow to specify other values than 50 to `dxx`.